<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLabelPropertyDialogBase</class>
 <widget class="QDialog" name="QgsLabelPropertyDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Label properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="mLabelTextLabel">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mLabelTextLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-319</y>
        <width>415</width>
        <height>619</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mLabelPropertiesLayout">
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mDisplayGroupBox">
         <property name="title">
          <string>Display</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Scale-based</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsSpinBox" name="mMinScaleSpinBox">
              <property name="prefix">
               <string>Min </string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="showClearButton" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsSpinBox" name="mMaxScaleSpinBox">
              <property name="prefix">
               <string>Max </string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="value">
               <number>10000000</number>
              </property>
              <property name="showClearButton" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="mShowLabelChkbx">
            <property name="text">
             <string>Show label</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="mAlwaysShowChkbx">
            <property name="toolTip">
             <string>Ignores priority and permits collisions/overlaps</string>
            </property>
            <property name="text">
             <string>Always show (exceptions above)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mFontGroupBox">
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="3">
           <widget class="QFontComboBox" name="mFontFamilyCmbBx">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Style</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QToolButton" name="mFontUnderlineBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <underline>true</underline>
               </font>
              </property>
              <property name="toolTip">
               <string>Underlined text</string>
              </property>
              <property name="text">
               <string>U</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="mFontStrikethroughBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <strikeout>true</strikeout>
               </font>
              </property>
              <property name="toolTip">
               <string>Strikeout text</string>
              </property>
              <property name="text">
               <string>S</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="mFontBoldBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Bold text
(data defined only, overrides Style)</string>
              </property>
              <property name="text">
               <string>B</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="mFontItalicBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="toolTip">
               <string>Italic text
(data defined only, overrides Style)</string>
              </property>
              <property name="text">
               <string>I</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="mFontSizeLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsDoubleSpinBox" name="mFontSizeSpinBox">
              <property name="specialValueText">
               <string>Default</string>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
              <property name="showClearButton" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="2">
           <widget class="QgsColorButtonV2" name="mFontColorButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="mFontStyleCmbBx">
            <property name="toolTip">
             <string>Available typeface styles</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mBufferGroupBox">
         <property name="title">
          <string>Buffer</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="mBufferSizeLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsDoubleSpinBox" name="mBufferSizeSpinBox">
              <property name="minimum">
               <double>-1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
              <property name="showClearButton" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QgsColorButtonV2" name="mBufferColorButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mPositionGroupBlox">
         <property name="title">
          <string>Position</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="mLabelDistanceLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Label distance</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsDoubleSpinBox" name="mLabelDistanceSpinBox">
            <property name="specialValueText">
             <string>Default</string>
            </property>
            <property name="minimum">
             <double>-1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999.000000000000000</double>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mXCoordLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>X Coordinate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsDoubleSpinBox" name="mXCoordSpinBox">
            <property name="specialValueText">
             <string>Default</string>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="mYCoordLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Y Coordinate</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QgsDoubleSpinBox" name="mYCoordSpinBox">
            <property name="specialValueText">
             <string>Default</string>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="mHaliLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Horizontal alignment</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="mHaliComboBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="mValiLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Vertical alignment</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="mValiComboBox"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="mRotationLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Rotation</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QgsDoubleSpinBox" name="mRotationSpinBox">
            <property name="specialValueText">
             <string>Default</string>
            </property>
            <property name="minimum">
             <double>-1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mLabelTextLineEdit</tabstop>
  <tabstop>mShowLabelChkbx</tabstop>
  <tabstop>mMinScaleSpinBox</tabstop>
  <tabstop>mMaxScaleSpinBox</tabstop>
  <tabstop>mAlwaysShowChkbx</tabstop>
  <tabstop>mFontFamilyCmbBx</tabstop>
  <tabstop>mFontStyleCmbBx</tabstop>
  <tabstop>mFontUnderlineBtn</tabstop>
  <tabstop>mFontStrikethroughBtn</tabstop>
  <tabstop>mFontBoldBtn</tabstop>
  <tabstop>mFontItalicBtn</tabstop>
  <tabstop>mFontSizeSpinBox</tabstop>
  <tabstop>mFontColorButton</tabstop>
  <tabstop>mBufferSizeSpinBox</tabstop>
  <tabstop>mBufferColorButton</tabstop>
  <tabstop>mLabelDistanceSpinBox</tabstop>
  <tabstop>mXCoordSpinBox</tabstop>
  <tabstop>mYCoordSpinBox</tabstop>
  <tabstop>mHaliComboBox</tabstop>
  <tabstop>mValiComboBox</tabstop>
  <tabstop>mRotationSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsLabelPropertyDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsLabelPropertyDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
